EIPS/eip-4844.md

```diff
@@ -322,6 +322,8 @@ The actual `data_fee` as calculated via `calc_data_fee` is deducted from the sen
 
 ### Networking
 
+Blob transactions are not automatically broadcast to peers. Instead, they are only announced using `NewPooledTransactionHashes` messages, and can then be manually requested via `GetPooledTransactions`.
```

Comment:
I wonder if this shall be moved to a separate EIP addressing Network layer only

Reply:
As this is required behavior for all nodes implementing 4844, it seems appropriate to keep it within the EIP.

Reply:
While I think there is no EIP editorial policy or guideline strictly prohibiting this, as far as I know, hence I think it's ok to stay same EIP. 

However, proper scoping could help improve readability and technical soundness for an EIP. For large EIPs like 4844, consider discuss among your coauthor to see if it make more sense to put network part of spec into the same core EIP or a separate one.

An example of split scoping: EIP-1559 depends on EIP-2718, EIP-2930 which depends on EIP-2929(Network).